projects/cdk/date-time/test/day.spec.ts

```diff
@@ -1046,6 +1046,17 @@ describe('TuiDay', () => {
                     });
                 });
             });
+
+            describe('valueOf returns', () => {
```

Comment:
I see that e2e tests have fallen, could you add unit tests for the cases that are on the pages?

Reply:
ye, will figure out